roles/cloud-bootstrap/tasks/create-users-projects.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Create bonnyci project
```

Comment:
the task names should be more generic, such as the `production project` or some such. Other implementers of this stuff may not want to call it bonnyci.